resources/home/dnanexus/eggd_artemis.py

```
@@ -242,7 +57,7 @@ def main(
                 f"with path {snv_path}"
             )
             multiqc_report = get_multiqc_report(
-                snv_path.split(",")[0], DX_PROJECT
+                snv_path.split(",")[0], project_id
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)